### ISSUE TITLE
Services missing after adding new user/roles

### ISSUE THREAD

**Issue by ascensionra** [dreamfactorysoftware/dsp-core#78]
I created a new user and some roles to give limited access to another user. After doing so all of my services are gone, although they still appear under API Docs, and are usable. I just can't see or edit them. I've had a quick peek at the debug logs and there were some errors:
* System error deleting swagger cache file:

It appears the cache files are still present in /opt/dreamfactory/platform/storage/.private/.cache/

**Comment by jstablesdf** [comment]
I have seen this error logged multiple times on DSPs that do not exhibit any problems. Are you creating and modifying these users and roles via the admin console UI or by API calls? Have you used the admin console UI to manually flush your system and swagger caches? Since the services still exist, I'm guessing you're using the admin console and this is a UI problem.

**Comment by ascensionra** [comment]
Correct, I’m using the UI, and I have used the UI to flush those caches.

**Comment by ascensionra** [comment]
Quick update. I created a brand new installation on a fresh Ubuntu Server VM. I created one service, an Oracle remote SQL service. I was able to access services, and the API Docs a few times, making changes to my connection string. The last time I attempted to update the connection string, i was no longer able to edit it. I used two other browsers, but right away I got the "You have no services" screen, although the items were still in the API Docs section. I flushed the system and swagger caches, but the behavior persists.

**Comment by jstablesdf** [comment]
We have been testing and have reproduced this issue somewhat by navigating back and forth between the Users and Apps tabs. However, the "loss" of services is not persistent. Upon reloading they re-appear. We are continuing to test and are trying to come up with concrete steps to reproduce so we can nail this one down.

**Comment by ascensionra** [comment]
Reloading the page, restarting the Apache server have no positive effect for me. Reloading puts me back to the login screen (regardless of whether I select 'remember me') and I get the same behavior. The login will process, and the progress bar will cycle through Config, App, and another (Admin?) that flashed by too quickly to read, but then leaves me with an empty content area (other than the buttons in top left and right.)

**Comment by ascensionra** [comment]
Any luck reproducing the behavior?

**Comment by jstablesdf** [comment]
No, we have not been able to force this to happen at all. Can you document steps to replicate? What did you name the service you were able to create before it disappeared? Anything else in the logs?

**Comment by ascensionra** [comment]
I'm trying to recreate, but at this point, I'm not even able to add a new service. I can fill in the form, but hitting create does nothing. I've got this behavior in IE,FF, and Chrome. I'm going to try on a second webserver to see if behavior persists.
